pandas/tests/frame/conftest.py

```diff
@@ -6,37 +6,37 @@
 
 
 @pytest.fixture
-def float_frame():
+def float_frame(seriesd):
```

Comment:
i can't see the benefit of making changes to this module or pandas/tests/frame/common.py in this PR.

Reply:
The alternative is to duplicate seriesd in the confest for excel. Would you rather do that and unify them in a separate PR?

Reply:
seriesd was not already defined as a fixture here, only in pandas/tests/groupby/conftest.py.

moving the seriesd fixture from pandas/tests/groupby/conftest.py to pandas/conftest.py does allow you to use it in pandas/tests/io/test_excel.py and would not conflict with anything here.

or leave it for a seperate PR.

Reply:
just created #26603 to make float_frame fixture available.

i'm trying to convince myself if we need a seriesd fixture at all. given it could be used to make float_frame composable, but i'm not sure this is necessary.

seriesd is just the values, so if we use them directly in any tests, we could just inject the float_frame fixture and use .values or to_numpy

As a DataFrame constructor accepts another DataFrame, we don't need seriesd to construct other fixtures.

we could just use something like

```
@pytest.fixture
def frame(float_frame):
    return float_frame[:10]
```

so for now i would not promote seriesd to top level conftest.

Reply:
i'm mixing-up examples..

should have been
```
@pytest.fixture
def frame():
    return DataFrame(float_frame, columns=list('DCBA'))
```

the other example is for a smaller float_frame fixture as required by pandas/tests/io/test_excel.py